bundles/net.tourbook.device/src/net/tourbook/device/sporttracks/FitLogSAXHandler.java

```diff
@@ -59,90 +59,92 @@
-   private static final String                  TAG_ACTIVITY_CATEGORY       = "Category";            //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_DURATION       = "Duration";            //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_DISTANCE       = "Distance";            //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_ELEVATION      = "Elevation";           //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_EQUIPMENT_ITEM = "EquipmentItem";       //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_HEART_RATE     = "HeartRate";           //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_LOCATION       = "Location";            //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY                        = "Activity";                            //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_CADENCE                = "Cadence";                             //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_CALORIES               = "Calories";                            //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_CATEGORY               = "Category";                            //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_DURATION               = "Duration";                            //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_DISTANCE               = "Distance";                            //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_ELEVATION              = "Elevation";                           //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_EQUIPMENT_ITEM         = "EquipmentItem";                       //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_HEART_RATE             = "HeartRate";                           //$NON-NLS-1$
+   private static final String                  TAG_ACTIVITY_LOCATION               = "Location";                            //$NON-NLS-1$
+   //
    private static final String                  TAG_ACTIVITY_CUSTOMTRACKS           = "CustomTracks";                        //$NON-NLS-1$
    private static final String                  TAG_ACTIVITY_CUSTOMTRACK            = "CustomTrack";                         //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_NAME           = "Name";                //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_NOTES          = "Notes";               //$NON-NLS-1$
-   private static final String                  TAG_ACTIVITY_POWER          = "Power";               //$NON-NLS-1$
+   //
+   private static final String                  TAG_ACTIVITY_NAME                   = "Name";                                //$NON-NLS-1$
```

Comment:
On the right side is the original code on the left side, the blue box is splitting the original code into 3 fragments and is inserted in the middle

![mt-fitlogsax](https://user-images.githubusercontent.com/1283445/104030759-29b77800-51cc-11eb-9ce0-405162be736e.png)

Reply:
I don't understand the issue here. you meant the static fields should be put in alphabetical order ?

Reply:
>I don't understand the issue here. you meant the static fields should be put in alphabetical order ?

I try to organize the fields by **grouping** and sorting to make them better readable but when I see the new fields (green background) they are spread over all existing fields without seeing that there is everywhere a sort order.

Mostly I use the alphabetical sort order for fields but not always, it depends.

![mt-fields](https://user-images.githubusercontent.com/1283445/104116640-9fc9f500-531a-11eb-94d2-583a6b022c0d.png)

Reply:
ok, I'll try to sort them by group then. my initial idea was to group the fields in such a way that we can find quickly the new one. but since you prefer alphabetial order in each group I can do that at least here.